avocado/core/job.py

```diff
@@ -169,12 +171,14 @@ def _make_test_runner(self):
 
     def _set_output_plugins(self):
         for key in self.args.__dict__:
-            if key.endswith('_result'):
-                result_class = getattr(self.args, key)
+            result_class = getattr(self.args, key)
+            try:
                 if issubclass(result_class, result.TestResult):
                     result_plugin = result_class(self.view,
                                                  self.args)
                     self.result_proxy.add_output_plugin(result_plugin)
+            except TypeError:
+                pass
```

Comment:
There's the possibility that the output plugin module itself is loadable, but the loader class is buggy, and for some reason it can't be loaded. In this case, the error won't get into the UI.

Now I would have to inject errors in loader classes and see what the behavior is so that we can figure out how to better handle such situations.